src/content/docs/en/developers/guides/canvas-badge-integration.mdx

```diff
@@ -0,0 +1,229 @@
+---
+section: developers
+date: Last Modified
+title: "Canvas & Badge Integration Guide"
+lang: "en"
+permalink: "developers/guides/canvas-badge-integration"
+excerpt: "This guide contains instructions on how to issue your own badge on Scroll Canvas."
+---
+
+import Aside from "../../../../../components/Aside.astro"
+import BadgeDetails from "../_images/badgeDetails.jpg"
+import BadgeLevels from "../_images/badgeLevels.png"
+import ClickToZoom from "../../../../../components/ClickToZoom.astro"
+import Columns from "../../../../../components/Columns.astro"
+import ToggleElement from "../../../../../components/ToggleElement.astro"
+
+We are thrilled to have you join us in building unique badges on Scroll Canvas, a product designed for ecosystem projects to interact with users in a more personal way on-chain. 
+
+## Background
+
+### Canvas
+
+Each Canvas is a `Profile` smart contract, minted by the users through the `ProfileRegistry` contract on Scroll. Canvas is not transferrable and is unique to each wallet.
```

Comment:
This page should link to https://github.com/scroll-tech/canvas-contracts somewhere, otherwise it's very hard for devs to reference what these contracts are.